packages/admin-ui/src/table/styles/table-row.styles.ts

```diff
@@ -1,8 +1,16 @@
 import { style } from '@vtex/admin-ui-core'
+import type { TableColumn } from '../types'
 
-export const rowBaseline = style({
-  display: 'table-row',
-  bg: '$primary',
-  textAlign: 'left',
-  borderBottom: '$neutral',
-})
+interface RowOptions {
+  columns: Array<TableColumn<any>>
+}
+export const rowBaseline = ({ columns }: RowOptions) =>
+  style({
+    display: 'grid',
+    bg: '$primary',
+    textAlign: 'left',
+    borderBottom: '$neutral',
+    gridTemplateColumns: columns
+      .map((column) => (column.width ? `${column.width}px` : '1fr'))
+      .join(' '),
```

Comment:
I've seen this pattern pop up a couple of times now. Maybe we should create a utility function to handle this grid template columns case?

Reply:
It still will change a little in the next PR's, but the gridTemplateColumns changes depending on the use case.

Reply:
I think [this is the other case](https://github.com/vtex/admin-ui/blob/dev/packages/admin-ui/src/grid/grid.tsx#L48-L50) where you saw this, but they are different. I'm not sure it's a pattern 🤔